Terminal.Gui/ConsoleDrivers/ConsoleDriver.cs

```diff
@@ -365,21 +365,19 @@ public void ClearContents ()
 	/// </summary>
 	public virtual bool SupportsTrueColor { get => true; }
 
-	bool _force16Colors = false;
-
-	// TODO: Make this a ConfiguationManager setting on Application
 	/// <summary>
-	/// Gets or sets whether the <see cref="ConsoleDriver"/> should use 16 colors instead of the default TrueColors.
+	/// Gets or sets whether the <see cref="ConsoleDriver"/> should use 16 colors instead of the default TrueColors. See <see cref="Application.Force16Colors"/>
+	/// to change this setting via <see cref="ConfigurationManager"/>.
 	/// </summary>
 	/// <remarks>
+	/// <para>
 	/// Will be forced to <see langword="true"/> if <see cref="ConsoleDriver.SupportsTrueColor"/> is  <see langword="false"/>, indicating
 	/// that the <see cref="ConsoleDriver"/> cannot support TrueColor.
+	/// </para>
 	/// </remarks>
-	public virtual bool Force16Colors {
-		get => _force16Colors || !SupportsTrueColor;
-		set {
-			_force16Colors = (value || !SupportsTrueColor);
-		}
+	internal virtual bool Force16Colors {
```

Comment:
Why is this no longer public? perhaps API users would like to pass `--oldcolors` to their app or some similar use case? The ability to set directly seems nice, unless that conflicts with the configuration manager somehow.  

Generally having more ways to do the same thing is better.

Reply:
Because `Application.Force16Colors` IS public (and configurable).

One other thing I'm trying to do is get rid of apps ever referencing `Driver.`.

Reply:
Cool, hadn't seen that it was an option on `Application` also.  🚀